docs/maintainers/growing-your-project.md

```diff
@@ -0,0 +1,80 @@
+sidebar_label: "Growing Your Project"
+keywords:
+  - "open source maintainers"
+  - "maintainers guides"
+  - "open source project success"
+  - "growing an open source project"
+---
+
+
+
+## When to Grow your Project
+
+When it comes to deciding on whether your open source project is ready to grow, consider asking yourself the following question:
+
+> Are there active community members who would be interested in progressing to an official role?
+
+If you have said yes, it's time to develop your growth strategies. If you haven't already, consider creating different teams for specific purposes. For example, you can create a docs team for documentation maintenance, a triage team for reviewing and approving issues, or a maintainer team to help you with overall project oversight. Once you have decided the types of teams you want to add to your project, it's time to start building the open source team of your dreams.
+
+## How to Identify New Contributors and Team Members
+
+When it comes to planning the growth strategies for your open source project, picking the right people will determine how successful your project will be. You can discover the right people for your project by starting with a [Repository Insight Page](../maintainers/maintainers-guide.md#creating-a-new-repository-insight-page). From there, you can use the Contributors dashboard to help you get a better understanding of your existing contributors.
+:::note
+
+If you need more assistance with finding new contributors and team members for your project, consider using our AI-powered chatbot, [StarSearch](../features/star-search.md#user-guide).
+
+:::
+
+Once you have found a reliable group of contributors, consider adding them to a [Contributor Insight Page](../maintainers/maintainers-guide.md#creating-a-new-contributor-insight-page). It can help you better understand their contributions to your project and the open source ecosystem. Here are some methods you can use to help you analyze its data:
+
+- Monitor contributors to see if they are active enough in your repositories to contribute as a team member.
+- Identify your top contributors to extend an invitation to join a team.
+- Identify active contributors in other repositories who may be interested in joining your team.
+
+If you need more tips on how to pick people that can help grow your project, check out our blog post, "[How to Build Your Open Source Dream Team](https://dev.to/opensauced/how-to-build-your-open-source-dream-team-a-guide-3i90)".
+
+## Recruiting Team Members and Contributors Outside of Your Project
+
+If you've looked at your project's [Repository Insight Page](../features/repo-insights.md) and realize there is no one that could help you, consider recruiting new contributors. To do that, create another Repository Insight Page with projects that are similar to yours. From there, determine which contributors suit your project, add them to a list, and monitor their progress.  
+
+:::tip
+
```

Comment:
The OSCR isn't on the Repo Insights page, so I'm not sure why it's being mentioned here.